### ISSUE TITLE
Move up / Move down action buttons - Tooltip should also show shortcut

### ISSUE THREAD

**Issue by porg** [WordPress/gutenberg#51647]
- Followup of #10455
  - ✅ Shortcut(s) are mentioned in the online documentation:
    - https://wordpress.org/documentation/article/keyboard-shortcuts-block-editor/#move-the-selected-blocks-up
    - https://wordpress.org/documentation/article/keyboard-shortcuts-block-editor/#move-the-selected-blocks-down
  - ❌ But there are no hints/cues in the UI

![WordPress Move block up down button tooltip shall also show corresponding shortcut](https://github.com/WordPress/gutenberg/assets/737143/7d03e214-2437-4457-84e1-6f6e780ee691)

## Proposal: The action button tooltips mention the shortcut in brackets
- Move up (⇧-⌥-⌘-T)
- Move down (⇧-⌥-⌘-Y)

## Related issue
- The shortcut should be shown in the tooltip whatever it will be in the future (not hardcoded, but reference), also when it is customizable
- Independent from this, the current shortcuts for "Move up/down" are only suited for QWERTY keyboards, see:
- #51645

**Comment by porg** [comment]
Hi,
I ping you, as you were originally involved:
@ZebulanStanphill as you opened #10455
@ntsekouras as you coded #23276

**Comment by joedolson** [comment]
This should also add the shortcut to the spoken message for screen reader users, so that both visual and non-visual users get this aid.

I suggest adding the shortcut at the end of the spoken message.

I also suggest wrapping the shortcut in parentheses, so it's visually a bit more clear that this is not part of the control's accessible name.
